roles/compliance/tasks/create_scap_report.yml

```diff
@@ -0,0 +1,28 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# This could be a loop over a map of scans. In fact, has to be include_tasks
+# Also extract the results and create the report.
+
+- name: Set base name
+  ansible.builtin.set_fact:
+    base_name: "{{ bzip_file.path[:-10] }}" # we expect files to end in ".xml.bzip2"
+
+- name: Unzip the file
+  ansible.builtin.shell: "bunzip2 -c {{ bzip_file.path }} > {{ base_name }}.xml"
```

Comment:
I see `shell` is used since we're rellying on `bunzip2 --stdout`

Reply:
Right.   The bzip files that are created by the oc compliance plugin have the format foo.bzip2.  This is, unfortunately not a recognized extension for bunzip2 (and I don't have any flag to pass into the compliance plugin to change it).

So, doing something simple like :" bunzip2 -c {{ bzip_file.path }} " would avoid the use of stdout, but would result in a warning message that the extension is unknown - and would produce a file which is  {{ bzip_file.path }}.out.  I suppose I could then rename that file to foo.xml.

This seems a somewhat circuitous way to get around simply using "shell" over "command".  It seems simpler and clearer to leave things as they are.

Reply:
If we don't want the shell module instead of command, I can always rename the file to a recognized extension ie. bz2 first before unzipping.   Maybe thats just the easiest way.